examples/README.md

```diff
@@ -1,10 +1,18 @@
 # Examples
 
-Demos to show key functionalities via notebooks and GUI applications, with [TorchScript](https://pytorch.org/docs/stable/jit.html) support.
+Demos to show key functionalities via notebooks and GUI applications, with [TorchScript](https://pytorch.org/docs/stable/jit.html) support (in future).
 
 Name convention: `data_method` or `data_method_lightn` for lightning
 
-Domain-specific development in three areas
+## Separate data and code
+
+To download by code or other ways to local directory rather than upload to GitHub.
+
+In general, we do not upload data to keep the repository size small. All data in examples (so far) are from the public domain so they are downloaded into local directories rather than uploaded to GitHub. This is done by setting `.gitignore` (see line 7). If we do want to share data, the data should be external to the PyKale repository (Google drive is popular nowadays).
```

Comment:
Maybe say that instructions for getting data are with the specific examples? I guess we may want to think about consistent ways of sharing data in future, if there is data that is essential for pykale to function. It's nice to have DOIs rather than links to commercial cloud storage as these have greater longevity. But don't let this slow down a merge.